### PR TITLE
Fix utils module not found

### PR DESCRIPTION
**Pull request by BeingGod**
Fix utils module not found bug when doing huggingface to megatron-lm model convert.

image

### CODE REVIEW

**Comment by BeingGod** [comment]
PTAL @SeaOfOcean

**Comment by SeaOfOcean** [comment]
@BeingGod thanks for the pr, you need to sign the license/cla first

**Comment by BeingGod** [comment]
> @BeingGod thanks for the pr, you need to sign the license/cla first

Done.

**Comment by BeingGod** [comment]
> @BeingGod thanks for the pr, you need to sign the license/cla first

It look like that I meet a problem. CLA signed page show that I have already signed CLA.
image

 What should I do now？

**Review by adoda — DISMISSED** [review]
LGTM

**Comment by adoda** [comment]
> > @BeingGod thanks for the pr, you need to sign the license/cla first
> 
> It looks like that I meet a problem. CLA signed page show that I have already signed CLA. image
> 
> What should I do now？

image

**Comment by BeingGod** [comment]
> > > @BeingGod thanks for the pr, you need to sign the license/cla first
> > 
> > 
> > It looks like that I meet a problem. CLA signed page show that I have already signed CLA. image
> > What should I do now？
> 
> image

Done.

**Comment by SeaOfOcean** [comment]
what‘s the docker image you use to reproduce the error? @BeingGod

**Comment by BeingGod** [comment]
> what‘s the docker image you use to reproduce the error? @BeingGod

My docker was built by https://github.com/alibaba/ChatLearn/blob/main/docker/torch/Dockerfile.torch2.3.0 .

File: chatlearn/tools/megatron_checkpoint_utils.py [hunk 1]
```diff
@@ -263,13 +263,13 @@ def load_module(self, name):
...
     if exist_checkpoint_util():
-        sys.meta_path.insert(-1, CheckpointUtilsImporter('tools.checkpoint.util', \
+        sys.meta_path.insert(0, CheckpointUtilsImporter('tools.checkpoint.util', \
             'tools.checkpoint.loader_megatron', 'tools.checkpoint.saver_megatron'))
         from tools.checkpoint import loader_megatron, saver_megatron # pylint: disable=unused-import
         from tools.checkpoint import util
         util.main()
     else:
-        sys.meta_path.insert(-1, CheckpointUtilsImporter('tools.checkpoint.convert', \
+        sys.meta_path.insert(0, CheckpointUtilsImporter('tools.checkpoint.convert', \
```
Comment (adoda) [inline chatlearn/tools/megatron_checkpoint_utils.py:272]: We could remove the CheckpointUtilsImporter from sys.meta_path after convert is compiled.

Comment (BeingGod) [reply chatlearn/tools/megatron_checkpoint_utils.py]: Ack, Thx.

**Review by haolin-nju — APPROVED** [review]
LGTM. Sorry for the late review.

**Review by adoda — APPROVED** [review]

**Comment by haolin-nju** [comment]
@BeingGod Congradulations! The PR can be automatically merged. Please click on `Squash and merge` and fill in commit msgs 🚀 🚀 🚀

**Pull request merged**